src/Startup.cs

```diff
@@ -76,9 +76,9 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILogger<
             {
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 app.UseHsts();
-            }
+                app.UseHttpsRedirection();
```

Comment:
Please take care of below warnings 

![image](https://user-images.githubusercontent.com/86330690/154134958-a0a3f42e-ec6c-41f7-bfd2-6d20301f98e5.png)